### DISCUSSION TITLE
How to start a DBus server application as a separate process from another process.

### DISCUSSION THREAD

**Discussion by vk-gst** [Kistler-Group/sdbus-cpp#200] [Q&A]
I have a DBus server application that uses sdbus-c++ bindings. Instead of this application to be run as a systemd service, I want to manually launch this server from an application X, as a separate process.  For eg. I want to do the `systemctl start server.service` in my C++ application X.

What would be the ideal way, to do this operation? I could think of using the command `system()`. Is there any other way to do this?

**Comment by sangelovic** [comment]
Launching a DBus server application as a separate process is just like launching any application as a separate process. So the question is quite general. You can find literature on this topic on the Internet...
